src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/TestFiles/RazorSemanticTokenInfoServiceTest/GetSemanticTokens_Razor_MultiLineCommentWithBlankLines_LF.semantic.txt

```diff
@@ -0,0 +1,10 @@
+//line,characterPos,length,tokenType,modifier
+0 0 1 razorCommentTransition 0
+0 1 1 razorCommentStar 0
+0 1 4 razorComment 0
+2 0 3 razorComment 0
+1 0 4 razorComment 0
+1 0 19 razorComment 0
+1 0 3 razorComment 0
+0 3 1 razorCommentStar 0
+0 1 1 razorCommentTransition 0
```

Comment:
Importantly, with the previous code, this test snapshot file and the above test snapshot  file were different, which is the bug this is fixing. All other test snapshots are unchanged because the rest is just perf work.